main/src/vtr_pose_graph/include/vtr_pose_graph/path/path.hpp

```diff
@@ -192,6 +195,12 @@ double Path<GraphT>::dist(unsigned seq_id) const {
   return distances_[seq_id];
 }
 
+template <class GraphT>
+int Path<GraphT>::query_terrain_type(unsigned seq_id) const {
+  int terrain_type = graph_->at(VertexId(0,seq_id))->GetTerrainType();
```

Comment:
Hard coding major id 0 is not ideal. Other teach passes can have greater than 0.

Reply:
Resolved by accessing sequence instead.